crypto/crypto.go

```diff
@@ -105,8 +105,8 @@ func Keccak512(data ...[]byte) []byte {
 }
 
 // CreateAddress creates an ethereum address given the bytes and the nonce
-func CreateAddress(b common.Address, nonce uint64) common.Address {
-	data, _ := rlp.EncodeToBytes([]interface{}{b, nonce})
+func CreateAddress(b common.Address, nonce uint64, code []byte) common.Address {
+	data, _ := rlp.EncodeToBytes([]interface{}{b, nonce, code})
```

Comment:
Should we call this arg (and in other fns) `data` instead of `code`? I think its called `data` everywhere else, so I'd like to stay consistent.

Reply:
It is called `code` in [evm.go](https://github.com/dominant-strategies/go-quai/blob/dev/core/vm/evm.go#L506)

Reply:
sounds good to me :shipit: